runtime/testy/stateManager.py

```diff
@@ -37,6 +40,7 @@ def getValue(self, key):
 
   def setValue(self, key, value):
     self.state[key] = value
+    print("NEW STATE:", self.state)
```

Comment:
remove. We want to avoid debugging print statements in code we push to master because

1. clutter
2. it should be debugged before we push